CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 0.4.8
 
 * Added support for `tf:suppress_diff` on primitive types ([#984](https://github.com/databrickslabs/terraform-provider-databricks/issues/984)).
+* Added support for release channels in SQL Endpoints configuration ([#1078])(https://github.com/databrickslabs/terraform-provider-databricks/pull/1078).
```

Comment:
SQL endpoints -> `databricks_sql_endpoint`, for consistency

Reply:
done